### PR TITLE
Adds Skub Suicide

### PR DESCRIPTION
**Pull request by ghost**
title
adds another dumb meme suicide, gibs the user instantly

https://gyazo.com/cd4f87f76f8d5fa0e5cfd81f79dc70e7
https://gyazo.com/6129396dbce0099e0cf7b0e7036eb6ec

### CODE REVIEW

**Comment by epochayur** [comment]
fuck you delete this PR now, skub killed my sister

**Comment by ExcessiveUseOfCobblestone** [comment]
what's the reason for the gibbing and does it destroy items on the person?

File: code/game/objects/items/miscellaneous.dm [hunk 1]
```diff
@@ -19,3 +19,9 @@
 	icon_state = "skub"
 	w_class = WEIGHT_CLASS_BULKY
 	attack_verb = list("skubbed")
+
+/obj/item/skub/suicide_act(mob/living/user)
+	user.visible_message("<span class='suicide'>[user] has declared themself as anti-skub! The skub tears them apart!")
+
+	user.gib()
+	playsound(loc, 'sound/items/eatfood.ogg', 50, 1, -1)
```
Comment (Cyberboss) [inline code/game/objects/items/miscellaneous.dm]: `loc` => `src`

Comment (Cyberboss) [reply code/game/objects/items/miscellaneous.dm]: return MANUAL_SUICIDE

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/objects/items/miscellaneous.dm (hunk 1 above)
Comment (Suicidalpickles) [reply code/game/objects/items/miscellaneous.dm]: changed those to the requested stuff, im awful at this tbh

**Comment by Suicidalpickles** [comment]
> 
> 
> what's the reason for the gibbing and does it destroy items on the person?

yeah it only leaves behind their glasses/boots/gloves/headset
also skub is just skub
one whom holds and opposes it is unworthy

**Comment by ExcessiveUseOfCobblestone** [comment]
in the future, branch off of your master. It'll save you a LOT of trouble.

File: code/game/objects/items/miscellaneous.dm [hunk 2]
```diff
@@ -19,3 +19,10 @@
 	icon_state = "skub"
 	w_class = WEIGHT_CLASS_BULKY
 	attack_verb = list("skubbed")
+
+/obj/item/skub/suicide_act(mob/living/user)
+	user.visible_message("<span class='suicide'>[user] has declared themself as anti-skub! The skub tears them apart!")
```
Comment (Cyberboss) [inline code/game/objects/items/miscellaneous.dm]: unclosed span

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/objects/items/miscellaneous.dm (hunk 2 above)
Comment (Suicidalpickles) [reply code/game/objects/items/miscellaneous.dm]: oops, adjusted that

**Comment by evsey9** [comment]
except bruteloss is boring

if you want their body back then go clone them, their head is right there.

**Comment by ExcessiveUseOfCobblestone** [comment]
> suicide
> cloning

hmmmm

**Comment by evsey9** [comment]
doesnt upgraded cloning let you clone them

**Comment by ExcessiveUseOfCobblestone** [comment]
if you can clone suicides it's a bug.

**Comment by evsey9** [comment]
just go goon's way and make suicided people still be able to be cloned
also what is the "If you don't want to play the game because you didn't get antag, then don't play at all." comment for

**Review by Cyberboss — APPROVED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
suicide is when someone doesn't want to play the round, so wanting them back in doesn't make much sense. It's why you can do whatever to them afterwards like toss them in the gibber. 

My bad for veering off, but to answer your question a large aspect of suiciding/not wanting to play the round is frankly not winning the antag lottery (I have never seen someone suicide after getting roundstart antag). Wasn't implying you specifically used it for such, sorry if it came across that way.

**Comment by octareenroon91** [comment]
drop an `unequip_everything()` in there somewhere

**Comment by YakumoChen** [comment]
>  (I have never seen someone suicide after getting roundstart antag).

you obviously weren't there for the round with the wizard who named himself "Suicide"

**Comment by Kagus** [comment]
There's a fellow who used to frequent Terry, whose personal goal in the game was to discover and experience every unique suicide animation/effect in the game. Pleasant fellow, all he wanted to do was kill himself.

Awfully unfortunate he picked Terry and its 3-hour-long(+) rounds, where he'd basically just have to sit around and chat with the other ghosties or play something else until next round when he could test out a new method of autoannihilation.

**Pull request merged**